crossbeam/xm_train.py

```diff
@@ -69,15 +72,30 @@ def main(argv) -> None:
 
     async def make_job(work_unit, **kwargs):
       args = deepcopy(executable_args)
-      args['config.save_dir'] = f'{save_dir}/{work_unit.work_unit_id}'
+      sweep_str_parts = []
+      for k, v in kwargs.items():
+        if k.startswith('config.'):
+          k = k[len('config.'):]
+        sweep_str_parts.append(f'{k}={v!r}')
+      sweep_str = ','.join(sweep_str_parts)
+      args['config.save_dir'] = f'{save_dir}/{work_unit.work_unit_id}_{sweep_str}'
```

Comment:
one minor issue: the 'save_dir/work_unit.work_unit_id' is unique enough to identify a folder, as the save_dir itself is formatted with xm job id. Do you still prefer adding the sweep_str as part of the folder name?

Reply:
yes because the folder name is what's shown on TensorBoard

Reply:
got it, then this looks perfect!